corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -341,18 +340,18 @@ hqDefine("app_manager/js/details/case_claim", function () {
                 if (["date", "daterange"].indexOf(searchProperties[i].input_) !== -1) {
                     appearance = searchProperties[i].input_;
                 }
-                var isMultiselect = searchProperties[i].input_ === "select",
-                    // The model supports multiple validation conditions, but we don't need the UI for it yet
-                    validation = searchProperties[i].validation[0];
+                // The model supports multiple validation conditions, but we don't need the UI for it yet
+                var validation = searchProperties[i].validation[0];
                 self.searchProperties.push(searchPropertyModel({
                     name: searchProperties[i].name,
-                    label: label,
-                    hint: hint,
+                    label: searchProperties[i].label[lang],
+                    hint: searchProperties[i].hint[lang] || "",
```

Comment:
Totally just pattern matching again, but I'm curious why `hint` defaults to blank string but `label` and `requiredMessage` don't - https://github.com/dimagi/commcare-hq/pull/31864 has made me suspicious of defaulting logic and falsy values.

Reply:
Good point, thanks for raising.  I just tried it out with `hint` set to the empty string for the currently active language, and with said language not defined.  It works fine without the fallback in both cases, so I'll remove it.